solarforecastarbiter/io/reference_observations/common.py

```diff
@@ -209,16 +209,54 @@ def create_observation(api, site, variable, extra_params=None, **kwargs):
         return created
 
 
+def _utcnow():
+    return pd.Timestamp.now(tz='UTC')
+
+
+def get_last_site_timestamp(api, site_observations, end):
+    """Get the last value timestamp from the API for a site
```

Comment:
There's nothing in the code that actually restricts this to observations from the same site, right? Maybe the argument should just be `observations`.